doc/source/user/numpy-for-matlab-users.rst

```diff
@@ -231,13 +134,39 @@ General Purpose Equivalents
      - short-circuiting logical OR operator (Python native operator);
        scalar arguments only
 
+   * - ::
+
+         a=1
+         if a==0
+             fprintf('a=0\n')
+         elseif a==1
+             fprintf('a=1\n')
+         end
+
+     - ::
+
+         a = 1
+         if a == 0:
+             print('a=0')
+         elif a == 1:
+             print('a=1')
+
+     - create an if-else statement to check if ``a`` is 0 or 1 and print result
+
    * - ``1*i``, ``1*j``,  ``1i``, ``1j``
      - ``1j``
      - complex numbers
 
    * - ``eps``
-     - ``np.spacing(1)``
-     - Distance between 1 and the nearest floating point number.
+     - ``np.finfo(float).eps`` or ``np.spacing(1)``
+     - Upper bound to relative error due to rounding in 64-bit floating point
+       arithmetic.
+
+   * - ``load data.mat``
+     - ``scipy.io.loadmat('data.mat')``
+     - load variables saved to the file ``data.mat`` note, if saving files from
+       MATLAB/Octave use a recent binary format. Python's ``loadmat`` will
```

Comment:
Punctuation seems off here

Reply:
```suggestion
       MATLAB/Octave use a recent binary format, Python's ``loadmat`` will
```

Reply:
updated the punctuation in discussion column